sources/osgViewer/Viewer.js

```diff
@@ -463,20 +462,21 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
             return P.reject();
         }
 
-        // reset position/orientation of hmd device
-        if ( !this._hmd.capabilities.hasPosition )
-            this._hmd.resetPose();
-
         if ( !this._hmd.capabilities.canPresent )
             return P.reject();
 
         if ( bool ) {
```

Comment:
While you're at it, could you change "bool" to something more explicit, like "presentVR"?